smsdk/client.py

```diff
@@ -338,12 +367,6 @@ def get_kpi_data_viz(
                 d_vars.append({"name": kpi, "aggregate": ["avg"]})
             kwargs["d_vars"] = d_vars
 
-        if i_vars:
```

Comment:
It seems like I can no longer pass in i_vars and time_selection via arguments if so this is a regression

Reply:
Addressed this issue.